backend/adminRouter.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+
+const Students = require('../models/students');
+
+const adminRouter = express.Router();
+
+dishRouter.use(bodyParser.json());
```

Comment:
Please do proper naming convention and while copy and pasting please rename all the variables.

```suggestion
adminRouter.use(bodyParser.json());
```